tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -59,6 +59,26 @@ arp/test_arp_dualtor.py::test_standby_unsolicited_neigh_learning[IPv4-active-sta
     conditions:
       - "asic_type in ['vs'] and https://github.com/sonic-net/sonic-mgmt/issues/17441"
 
+arp/test_arp_extended.py::test_arp_garp_enabled:
+  xfail:
+    reason: "xfail for IPv6-only topologies, with issue it try to parse with IPv4 style (https://github.com/sonic-net/sonic-mgmt/issues/19904)"
+    conditions_logical_operator: and
+    conditions:
+      - "'-v6-' in topo_name"
+      - "https://github.com/sonic-net/sonic-mgmt/issues/19904"
```

Comment:
move these 2 things in a single line, so we can show the correlations

Reply:
same for the rest.

Reply:
Changed. Thanks